composer.json

```diff
@@ -27,7 +27,8 @@
 	},
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.3",
-		"phpmd/phpmd": "~2.3"
+		"phpmd/phpmd": "~2.3",
+		"phpunit/phpunit": "~4.8"
```

Comment:
Should be `~3.7|~4.0` in my opinion, because what we are doing in this component works with all these versions.